tests/testthat/_snaps/xml_find.md

```diff
@@ -2,6 +2,10 @@
 
     Element at path `//z` must be a number, not a <xml_missing> object.
 
+# xml_find_int errors with non integer results
```

Comment:
Looks like this only caught the first error. But this issue is also present in the other usages in this file -- I would relegate to a broader follow-up issue.